sources/tests/workflow-docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+
+services:
+
+  zeebe-test:
+    image: camunda/zeebe:${CAMUNDA_PLATFORM_VERSION:-8.1.1}
+    container_name: zeebe-test
+    environment:
+      - ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_CLASSNAME=io.camunda.zeebe.exporter.ElasticsearchExporter
+      - ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_URL=http://elasticsearch-test:9200
+      - ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_BULK_SIZE=1
+      - ZEEBE_BROKER_DATA_DISKUSAGECOMMANDWATERMARK=0.998
+      - ZEEBE_BROKER_DATA_DISKUSAGEREPLICATIONWATERMARK=0.999
+      - "JAVA_TOOL_OPTIONS=-Xms256m -Xmx256m"
+    restart: always
+    networks:
+      - local-network-tests
+    depends_on:
+      - elasticsearch-test
+
+  elasticsearch-test:
+    image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTIC_VERSION:-7.17.5}
+    container_name: elasticsearch-test
+    environment:
+      - bootstrap.memory_lock=true
+      - discovery.type=single-node
+      - xpack.security.enabled=false
+      - cluster.routing.allocation.disk.threshold_enabled=false
+      - "ES_JAVA_OPTS=-Xms256m -Xmx256m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    restart: always
+    networks:
+      - local-network-tests
+  
+  workflow:
+    image: workflow-${PROJECT_NAME}:${BUILD_REF}-${BUILD_ID}
+    container_name: workers-${PROJECT_NAME}-${BUILD_REF}-${BUILD_ID}
+    build:
+      context: ../packages/backend
+      dockerfile: ./workflow/Dockerfile.dev
+    environment:
+      - NODE_ENV=docker
+      - ZEEBE_ADDRESS=zeebe-test:26500
+      - ZEEBE_CLIENT_MAX_RETRIES=${ZEEBE_CLIENT_MAX_RETRIES}
+    networks:
+      - local-network-tests
+    restart: always
+    depends_on:
+      - zeebe-test
+
+
+
+networks:
+  local-network-tests:
```

Comment:
This is the same name as the network used for the docker-compose under tests.
We can use a different name or just let the docker-compose create the default one by itself.